### PR TITLE
Update Hebrew translation

### PR DESCRIPTION
**Pull request by orgads**
* Replace הבניית חשבון to התקנת חשבון
* Add some missing translations

### CODE REVIEW

File: he/psi_he.ts [hunk 1]
```diff
@@ -2584,7 +2584,7 @@ Proceed?</source>
         <location filename="../psi/src/disco.ui" line="28"/>
         <source>ToolBar will be placed here</source>
         <comment>Don&apos;t translate this string</comment>
```
Comment (GreenLunar) [inline he/psi_he.ts]: Comment: Don't translate this string

`<comment>Don&apos;t translate this string</comment>`

File: he/psi_he.ts [hunk 2]
```diff
@@ -2584,7 +2584,7 @@ Proceed?</source>
...
-        <translation type="unfinished"></translation>
+        <translation>סרגל הכלים ימוקם כאן</translation>
```
Comment (GreenLunar) [inline he/psi_he.ts]: Comment: Don't translate this string

<comment>Don&apos;t translate this string</comment>

File: he/psi_he.ts [hunk 3]
```diff
@@ -9139,7 +9139,7 @@ This may mean you are already running another instance of Psi. You may experienc
     <message>
         <location filename="../psi/src/psicontactlistviewdelegate.cpp" line="149"/>
         <source>%1 (%2)</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (GreenLunar) [inline he/psi_he.ts]: Maybe we should add RLM character for proper alignment.

%1 RLM(%2)
RLM%1 RLM(%2)
RLM%1 RLM(%2)RLM

What do you think?

Comment (orgads) [inline he/psi_he.ts]: I don't have the source currently. I trust you on that.

**Comment by GreenLunar** [comment]
I will look into it later.

**Pull request merged**